### PR TITLE
Add scarlet to parse events into notes

### PR DESCRIPTION
**Pull request by Synesso**

### CODE REVIEW

**Comment by jmateoac** [comment]
Current dependencies on/for this PR:
* main
  * **PR #10** <a href="https://app.graphite.dev/github/pr/jmateoac/plasma/10" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jmateoac/plasma/10?utm_source=stack-comment).

File: app/src/main/kotlin/social/plasma/ui/feed/FeedViewModel.kt [hunk 1]
```diff
@@ -28,19 +26,17 @@ class FeedViewModel @Inject constructor(
 
     // TODO only subscribe to relays once during the entire app cycle
     // https://github.com/nostr-protocol/nips/blob/master/01.md#other-notes
-    private val relaySubscription =
-        relays.subscribe("wss://relay.damus.io").filter { it.contains("kind\":1") }
+    private val relaySubscription: Flow<Note> = runBlocking {
```
Comment (Synesso) [inline app/src/main/kotlin/social/plasma/ui/feed/FeedViewModel.kt:29]: Is this `runBlocking` appropriate, @jmateoac ?

**Review by jmateoac — APPROVED** [review]
Amazing!

File: app/src/main/kotlin/social/plasma/ui/feed/FeedViewModel.kt (hunk 1 above)
Comment (jmateoac) [reply app/src/main/kotlin/social/plasma/ui/feed/FeedViewModel.kt]: It blocks the UI thread which can lead to ANRs
Just pushed a commit that splits the subscription and does it on the IO thread pool

**Pull request merged**